code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -259,13 +261,14 @@
 	)
 
 /datum/gas_reaction/h2fire/react(datum/gas_mixture/air, datum/holder)
+	. = VOLATILE_REACTION
```

Comment:
Rather than copypaste this on every gas.
It's probably better to have a `var/gas_flags` on `/datum/gas_reaction`
And then assign volatility on parent call

Reply:
Gas reactions don't call parent and some of these volatile flags are conditional. Not sure it would be worth it.

Reply:
ok but Im saying you could simply add `. = ..()` and then in the parent call just change the behaviour based on a flag you add to each subtype.